src/python/bot/fuzzers/syzkaller/fuzzer.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""syzkaller fuzzer."""
+from __future__ import absolute_import
+from base import utils
+from bot.fuzzers import builtin
+from bot.fuzzers import utils as fuzzer_utils
+from bot.fuzzers.syzkaller import constants
+from bot.fuzzers.syzkaller import generate_config
+from builtins import object
+from system import environment
+from system import new_process
+import copy
+import re
+import tempfile
+
+# Regex to find testcase path from a crash.
+KASAN_CRASH_TESTCASE_REGEX = (r'.*Test unit written to\s*'
+                              r'(Read|Write) of .*')
+
+
+def get_arguments(fuzzer_path):
+  """Get arguments for a given fuzz target."""
+  del fuzzer_path
+  build_dir = environment.get_value('BUILD_DIR')
+  device_serial = environment.get_value('ANDROID_SERIAL')
+  json_config_path = '/tmp/' + device_serial + '/config.json'
+  generate_config.run(
+      serial=device_serial,
+      work_dir_path='/tmp/syzkaller',
```

Comment:
where do these paths come from? can we put them in a constant where it makes sense to?

Reply:
Done.